build.gradle

```diff
@@ -800,6 +803,14 @@ project('spring-integration-xmpp') {
 	}
 }
 
+project('spring-integration-zmq') {
```

Comment:
What's wrong to call the module as `zeromq` ?
Bear in mind it is going to be a public artifact for dependencies.

@garyrussell , your opinion?

Thanks